components/Footer/Footer.tsx

```diff
@@ -1,8 +1,10 @@
 export default function Footer() {
+  const currentYear = new Date().getFullYear();
+  console.log(currentYear);
   return (
     <footer>
-      <div className="container">
-        <p>all works are copyright Oly Su 2022</p>
+      <div className="container mb-6">
```

Comment:
Might be good to add margin-top as well, so there will always be a gap between footer and the component above.

```suggestion
      <div className="container my-6">
```